src/test/java/org/opensearch/neuralsearch/common/BaseNeuralSearchIT.java

```diff
@@ -491,4 +491,25 @@ protected static class KNNFieldConfig {
         private final Integer dimension;
         private final SpaceType spaceType;
     }
+
+    @SneakyThrows
+    protected void deleteModel(String modelId) {
+        makeRequest(
+            client(),
+            "POST",
+            String.format(LOCALE, "/_plugins/_ml/models/%s/_undeploy", modelId),
```

Comment:
Can we add constants for the fields in path and for Kibana ?

Reply:
ack